### PR TITLE
LPS-78396 Update property description

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPS-78396

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
Hey @brianikim, could you please have a look to the PR? Thx

**Comment by brianikim** [comment]
LGTM.

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -2220,6 +2220,13 @@
     # The following variables can be used: ${liferay:screenName} and
     # ${liferay:userId}.
     #
+    # ${liferay:userId} should only be used if your portlet needs the userId as
+    # a parameter. For example, if the portlet URL is in the form:
+    # /web/SCREEN_NAME/profile/-/PORTLET_NAMESPACE/USER_ID, setting
+    # users.profile.friendly.url =
+    # /web/${liferay:screenName}/profile/-/PORTLET_NAMESPACE/${liferay:userId}
+    # will allow you to provide a direct link to that portlet.
```
Comment (andrea-ale-sbarra) [inline portal-impl/src/portal.properties:2228]: @JamesAGarcia can you please have a look at this?
Thank you!

**Review by andrea-ale-sbarra — COMMENTED** [review]
Waiting for @JamesAGarcia before forwarding this.
Thank you!

File: portal-impl/src/portal.properties (hunk 1 above)
Comment (JamesAGarcia) [reply portal-impl/src/portal.properties:2228]: Hello there @andrea-ale-sbarra and @smottal! These are my first thoughts:

```
    # Only use ${liferay:userId} if your portlet needs the userId as a parameter.
    # For example, if the portlet URL uses this form,
    # /web/SCREEN_NAME/profile/-/PORTLET_NAMESPACE/USER_ID,
    # setting users.profile.friendly.url =
    # /web/${liferay:screenName}/profile/-/PORTLET_NAMESPACE/${liferay:userId}
    # provides a direct link to the portlet.
```

However, I am not sure I quite follow the meaning of the above text, especially when taken with the existing text 🤔 

```
    # Set the friendly URL to a user's profile page. If none is specified, the
    # portal will query the user's first public page at runtime.
    #
    # The following variables can be used: ${liferay:screenName} and
    # ${liferay:userId}.
```

Taken together, it seems like this section is saying...
* Use `users.profile.friendly.url` to set a friendly URL for accessing a user's profile page.
* When defining the URL, you can use the `${liferay:screenName}` or `${liferay:userId}` variables.
* Generally, you only need to use the `${liferay:screenName}` variable (e.g., the default value `users.profile.friendly.url=/web/${liferay:screenName}/profile`)
* You should only use `${liferay:userId}` if you have a 'portlet' (?) that needs the ID.

Question: What 'portlet' is the new note talking about? Is it a custom portlet? The referent for these phrases is unclear to me: "if your _**portlet**_ needs..." and "...will allow you to provide a direct link to that portlet." 

Ultimately, the context for the new text is unclear to me. Any additional clarification you can give would be helpful.

Comment (andrea-ale-sbarra) [reply portal-impl/src/portal.properties:2228]: Yes the portlet a custom portlet and looking at the message I think that the most clear message would be

```suggestion
    # Only use ${liferay:userId} if your portlet needs the userId as a parameter. 
    # For example, if your portlet URL uses this form, 
    # /web/SCREEN_NAME/profile/-/PORTLET_NAMESPACE/USER_ID, 
    # setting users.profile.friendly.url = 
    # /web/${liferay:screenName}/profile/-/PORTLET_NAMESPACE/${liferay:userId} 
    # provides a direct link to your portlet.
```

**Comment by smottal** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:test:stable

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**